### ISSUE TITLE
Binding mouse buttons ?

### ISSUE THREAD

**Issue by LoganTann** [CCDirectLink/input-api#1]
Hello, I have a mouse with five buttons, and I would like to assign the 4th and 5th button. 

Is it technically feasible ?

**Comment by 2767mr** [comment]
Yes, it's feasible and should probably be included in this mod as well.
Here is one mod that has it: https://github.com/dmitmel/crosscode-tweak-pack#bindable-mouse-buttons-qol
